Backend/src/main/java/RunBackEnd.java

```diff
@@ -21,7 +22,7 @@ public static void main(String[] args) throws InterruptedException {
             run("mQinstance");
             services = new ServicesType[]{ServicesType.user,ServicesType.moderator};
             run("controller");
-
+            new RedisConnection();

```

Comment:
Could you add it to the service controller of each micro service instead of in yhe runBafkEnd, as the connections should be for each micro service and not one for the entire apllication, you will find an init method in the services classes

Reply:
Yes, i was only testing with one connection and a single redis server, I will change this in the next PR when i add clustering to redis servers